src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/Internal/Win32.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
Is this needed?
In the `csproj` (below) nullability is turned on.

Reply:
Missed that this file is located under `Microsoft.Extensions.Hosting.WindowsServices`, thanks